spec/models/group_entity_spec.rb

```diff
@@ -1,5 +1,3 @@
-# frozen_string_literal: true
-
 require 'rails_helper'
```

Comment:
- Hi, Can you provide the unit tests for all the most important components, also integration tests for all the most important components? I would advise you to check the test if it has passed before you ask for a review. 

![Capture](https://github.com/fitz95/Budget_App/assets/63972670/c65097ba-ac1e-49a3-94d2-6f662aa63513)
NB: It's par of the project requirements

Reply:
@OLIPLICHE  Thanks for the review I will do as you asked